lib/api/src/externals/function.rs

```diff
@@ -573,7 +582,56 @@ impl Function {
     pub fn call(&self, params: &[Val]) -> Result<Box<[Val]>, RuntimeError> {
         if let Some(trampoline) = self.exported.vm_function.call_trampoline {
             let mut results = vec![Val::null(); self.result_arity()];
-            self.call_wasm(trampoline, params, &mut results)?;
+            self.call_wasm(trampoline, params, &mut results, false)?;
+            return Ok(results.into_boxed_slice());
+        }
+
+        unimplemented!("The function definition isn't supported for the moment");
+    }
+
+    /// Call the `Function` function (unchecked).
```

Comment:
This comment doesn't explain what the difference between `call` and `call_unchecked`. Yes, we need to wrap `call_unchecked` in `Store::catch_traps`, OK, but it doesn't explain why I should `call` rather than `call_unchecked` and vice versa. Can you update the comment please :-)?